docs/src/examples/components/Dropdown/Types/index.tsx

```diff
@@ -4,6 +4,11 @@ import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
 
 const Types = () => (
   <ExampleSection title="Types">
+    <ComponentExample
+      title="Single Selection"
+      description="A dropdown with single selection."
+      examplePath="components/Dropdown/Types/DropdownExampleSingleSelection.shorthand"
```

Comment:
```suggestion
      examplePath="components/Dropdown/Types/DropdownExampleSingleSelection"
```

We don't need `.shorthand` there

Reply:
fixed, thanks!